board_size:
- 8
- 11
calibration_date: '2025-12-15 14:11:36'
camera_matrix:
  cx: 319.07595214284834
  cy: 255.5166207673894
  fx: 784.6866524961796
  fy: 784.6809717665531
camera_matrix_full:
- - 784.6866524961796
  - 0.0
  - 319.07595214284834
- - 0.0
  - 784.6809717665531
  - 255.5166207673894
- - 0.0
  - 0.0
  - 1.0
distortion_coefficients:
- - -0.2674673344331669
  - 2.941623964288034
  - 0.015358190117308844
  - -0.01253113802326706
  - -13.490343588740046
num_images: 40
square_size: 1.5
